openmetadata-ui/src/main/resources/ui/src/components/AddTags/add-tags.component.tsx

```diff
@@ -33,10 +34,14 @@ export const AddTags = ({
 
   const fetchTags = async () => {
     setIsTagLoading(true);
-    const tags = await getAllTagsForOptions();
-    setTagList(tags.map((t) => t.fullyQualifiedName || t.name));
-
-    setIsTagLoading(false);
+    try {
+      const tags = await getAllTagsForOptions();
+      setTagList(tags.map((t) => t.fullyQualifiedName || t.name));
+    } catch (error) {
```

Comment:
@ShaileshParmar11 Do we really need a component here?
I feel it should be handled in the page. Check if we are re-using it anywhere else, If not remove the component itself

Reply:
Yes we are using the component in create glossary and create glossary term page